src/Coordination/KeeperStorage.cpp

```diff
@@ -363,7 +363,7 @@ void KeeperStorage::UncommittedState::applyDelta(const Delta & delta)
             {
                 assert(my_node);
                 my_node->invalidateDigestCache();
-                operation.update_fn(*node);
+                operation.update_fn(*my_node);
```

Comment:
we can simply remove explicit reference bindings and leave `[&]`?

https://godbolt.org/z/W6h14jeeE

Reply:
Yep, but I guess the reason was to explicitly show what should be binded.

Reply:
So what would you prefer, simply `auto & [node, acls, last_applied_zxid] = nodes.at(delta.path)` inside lambda?

Reply:
yeah